public/static/docs/command-reference/get.md

```diff
@@ -59,7 +59,7 @@ name.
   it.
 
 - `--rev` - specific
-  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  [Git reference](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
```

Comment:
not sure 100% about this, but it feels we better keep revision (even for the sake of being consistent with the param name), also this doc is probably better suited - https://git-scm.com/docs/revisions

Reply:
applies to all other cases in this PR

Reply:
Yes, https://git-scm.com/docs/revisions is much better. Idk why I didn't find that page before...

Reply:
OK! I finally reviewed the use of this and related terms throughout the docs in 5a08d37f, using this criteria: Use "version" for data or DVC-tracked files, "revision" (and that link) in the context of pure Git/ source code, and "reference" sparingly when actually talking about references (to Git revisions).
It's an interesting question. See also https://github.com/iterative/dvc/pull/3244#pullrequestreview-348990111

p.s. big change, sorry... I realize this review takeover is taking too long. Will try to focus more now.